### PR TITLE
Ensure prettier recommended eslint rules are being applied

### PR DESCRIPTION
**Pull request by sareh**
- Ensures prettier's recommended eslint rules are being applied.  (in `.eslintrc`)
- Turns off the rule `react/jsx-one-expression-per-line` (in `.eslintrc`)
- Fixes all these eslint issues across all relevant js, jsx, json files. 

* ~[ ] Fixes https://github.com/bbc/simorgh/issues/NUMBER~
* ~[ ] Tests added for new features~

* [ ] Test engineer approval

### CODE REVIEW

**Review by DemonusPC — DISMISSED** [review]

File: src/app/components/Headline/index.jsx [hunk 1]
```diff
@@ -17,11 +17,7 @@ const Headline = ({ blocks }) => {
     return null;
   }
 
-  return (
-    <StyledHeadline>
-      {text}
-    </StyledHeadline>
-  );
+  return <StyledHeadline>{text}</StyledHeadline>;
```
Comment (sareh) [inline src/app/components/Headline/index.jsx:20]: The duplication highlighted here by CodeClimate will be addressed in a separate PR.

**Review by DemonusPC — APPROVED** [review]

**Review by jamesbhobbs — APPROVED** [review]
